documentation/src/main/asciidoc/userguide/chapters/multitenancy/MultiTenancy.adoc

```diff
@@ -46,17 +46,6 @@ Using this approach, we would have a distinct JDBC Connection pool per-tenant wh
 * Connections could point to the database itself (using some default schema) but the Connections would be altered using the SQL `SET SCHEMA` (or similar) command.
 Using this approach, we would have a single JDBC Connection pool for use to service all tenants, but before using the Connection, it would be altered to reference the schema named by the "tenant identifier" associated with the currently logged in user.
 
-[[multitenacy-discriminator]]
```

Comment:
Not sure why we are removing this section.  This is still a known/valid form of multi-tenancy.  This section is just discussing the various forms of multi-tenancy.  Just note below that support for this form is not (yet) implemented